util/src/main/java/io/kubernetes/client/util/ModelMapper.java

```diff
@@ -114,8 +115,34 @@ public static void addModelMap(String group, String version, String kind, Class<
    */
   public static void addModelMap(
       String group, String version, String kind, String resourceNamePlural, Class<?> clazz) {
+    // TODO(yue9944882): consistency between bi-directional maps
```

Comment:
I would just add a wrapper class that contains two `ConcurrentHashMap` and implements `Map` wdyt?